documentation-site/examples/layout-grid/custom-css-unit.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+import * as React from 'react';
+import {useStyletron} from 'baseui';
+import {Grid, Cell} from 'baseui/layout-grid';
+
+export default () => (
+  <Outer>
+    <Grid
+      gridUnit="em"
+      gridGutters={[1, 2, 2]}
+      gridMargins={[1, 2, 4]}
+    >
+      <Cell gridUnit="em">
```

Comment:
`gridUnit` only needs to be set on `Grid` since it passes it through to children.